packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -60,6 +71,16 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
         },
       },
 
+      /**
+       * The delay in milliseconds before the tooltip
+       * is opened on hover, when not in manual mode.
+       * On focus, the tooltip is opened immediately.
+       */
+      delay: {
+        type: Number,
+        value: () => defaultDelay,
```

Comment:
Should we skip adding the default delay as the `delay` value for an individual tooltip instance and just use the default as an internal fallback value in case `delay` isn't set? Kind of similarly as in https://github.com/vaadin/web-components/pull/4402

This would also enable changing the default value (with `setDefaultDelay`) after `<vaadin-tooltip>`s have been instantiated so there would be less chance for timing issues.

Reply:
Good point, I will update the PR.

Reply:
Done.